### ISSUE TITLE
Vorschlag: Neue Statuskategorie zum Nachverfolgen von eingereichten Anzeigen

### ISSUE THREAD

**Issue by sakudo** [weg-li/weg-li#680]
Danke für das tolle Weg.li!
Ich würde eine weitere Statuskategorie begrüßen, mit der man den Erfolg der Anzeige dokumentieren könnte.

Problem: Ich habe festgestellt, dass manche Anzeigen erst auf Nachfrage bei den zuständigen Behörden aufgenommen werden. Man schreibt also z. B. eine Nachfrage nach einer Woche, und erhält dann manchmal eine Antwort mit einem vermeintlichen Grund, warum man die Anzeige nicht schreiben kann. Wenn man sich dann hartnäckig gibt und das dargelegte, vermeintliche Argument widerlegt, kommt es dann zu einer Anzeige.

Um diesen Prozess nachzuverfolgen wäre ein zusätzlicher Status "Erfolgsstatus" hilfreich:

- unbekannt (default, wenn Anzeige eingereicht)
- na (wenn Anzeige nicht eingereicht)
- Nachgefrage erfolgt
- Ablehnung mit nachvollziehbarer sachlicher Begründung betr. Tatbestand
- Ablehnung mit Ausrede / Opportunitätsprinzip
- Ablehnung wegen Formfehler
- Dienstaufsichtsbeschwerde eingereicht
- Erfolg 

Natürlich sind die Kategorien nicht absolut sicher oder nachzuprüfen, aber für den Anzeigenerstatter doch recht praktisch. Eine gewisse Auswertung von weg.li könnte dennoch interessant sein.

(Leider verstehe ich zu wenig von der Programmierung, so dass ich das nicht selbst realisieren kann.)

**Comment by phoet** [comment]
Was du vorschlägst ist quasi eine Möglichkeit Tags zu verteilen. Das gab es schon mal irgendwo als Feature wunsch. Das wäre auf jeden Fall eine sinnvolle Ergänzung der Funktionalität, auch wenn ich ihr keine hohe Prio einräume.

**Comment by sakudo** [comment]
Ja, mit einer Tag-Funktionalität (und möglichst einer entsprechenden Filterung) könnte man das gut abbilden.

Die Frage ist, ob die Tags fix vorgegeben sind (Vorteil: übergeordnete Auswertung könnte ggf. möglich sein), oder der Benutzer könnte selbst die Tags definieren. Ersteres wäre wohl einfacher und auch attraktiver, da es mehr zum Benutzen einlädt.

In der Tat: keine hohe Prio, aber dennoch praktisch, da ich z. Z. eine private Calc-Liste führe, die genau dies leistet.

**Comment by woderkant** [comment]
Moin sakudo,
 
(weiß leider nicht, wie ich dich hier direkt anschreiben kann)
> Problem: Ich habe festgestellt, dass manche Anzeigen erst auf Nachfrage bei den zuständigen Behörden aufgenommen werden. Man schreibt also z. B. eine Nachfrage nach einer Woche, und erhält dann manchmal eine Antwort mit einem vermeintlichen Grund, warum man die Anzeige nicht schreiben kann. Wenn man sich dann hartnäckig gibt und das dargelegte, vermeintliche Argument widerlegt, kommt es dann zu einer Anzeige.

Wie genau stellst du Nachfragen? Die zuständige Behörde lehnt Nachfragen zu Anzeigen grundsätzlich ab und beruft sich auf den Datenschutz.

Schreib mir gerne direkt an wegli@mailbox.org
Danke.

**Comment by sakudo** [comment]
Email schreiben oder anrufen. Beim KVR in München bin ich meistens weitergekommen. Ist zäh. Kostet halt Nerven.

Beim Gegenargument "Datenschutz" würde ich gleich den Spieß umkehren und darauf aufmerksam machen, dass man gem. DSGVO ein Recht hat, über seine eigene Daten und deren Verwendung informiert zu sein, sprich, wenn der eigene Name als Zeuge in einer Anzeige verwendet wird, dann kann ich auch die Anzeige (mit geschwärztem Namen des Angezeigten) erhalten.

Sonst halt mal ausprobieren und eine Auskunftsanfrage gem. DSGVO stellen. Alles mühsam, ich weiß...
